ssds/blobstore/s3.py

```diff
@@ -73,6 +73,21 @@ def put(self, data: bytes):
         blob = self._s3_bucket.Object(self.key)
         blob.upload_fileobj(io.BytesIO(data))
 
+    def copy_from(self, src_blob: "S3Blob"):
```

Comment:
Same.  Should this only copy between s3 buckets or should this be cloud agnostic?